Leanplum/Protocol/LPCacheProtocol.h

```diff
@@ -0,0 +1,36 @@
+//
+//  LPCacheProtocol.h
+//  Leanplum
+//
+//  Created by Grace on 7/29/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class LPVariable;
+@class LPRegion;
+@class LPMessage;
+@class LPVariant;
+@class LPUpdateRule;
+@class LPEventRule;
+@class LPVariantDebugInfo;
+
+@protocol LPCacheProtocol <NSObject>
```

Comment:
in obj-c we typically use the "..ing" naming convention.

https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html#//apple_ref/doc/uid/20001281-1002242-BBCIJGDB

Let's call this `LPCaching`